.github/workflows/run-checker-daily.yml

```diff
@@ -207,7 +207,7 @@ jobs:
     - name: checkout fuzz/corpora submodule
       run: git submodule update --init --depth 1 fuzz/corpora
     - name: config
-      run: ./config --with-rand-seed=none enable-jitter --with-jitter-include=jitter/ --with-jitter-lib=jitter/ -DOPENSSL_DEFAULT_SEED_SRC='"JITTER"' && perl configdata.pm --dump
+      run: ./config --with-rand-seed=none enable-jitter --with-jitter-include=jitter/ --with-jitter-lib=jitter/ -DOPENSSL_DEFAULT_SEED_SRC='\"JITTER\"' && perl configdata.pm --dump
```

Comment:
Given that users will have to define this as well, might it be better to allow OPENSSL_DEFAULT_SEED_SRC to be defined without quotes, and just stringify it in the code? i.e., in rand_lib.c:

```
#define STR(x) #x
...
name = STR(OPENSSL_DEFAULT_SEED_SRC);
```

Reply:
Ok will fix it to not require quotes on the argument to configure.

Reply:
Used existing `OPENSSL_MSTR` define for this